### DISCUSSION TITLE
Модальное окно - позиция по центру

### DISCUSSION THREAD

**Discussion by vadzab** [themesberg/flowbite-vue#274] [General]
Привет.
Заметил что модальное окно показывается не по центру. Это так задумано или в классе ошибка?
Код из [FwbModal.vue](https://github.com/themesberg/flowbite-vue/blob/main/src/components/FwbModal/FwbModal.vue)
```
11:      <div
12:        :class="`${modalSizeClasses[size]}`"
13:        class="relative p-4 w-full h-full" 
14:      >
```

Чтобы было по центру нужно поменять `h-full` -> `max-h-full`. Если так задумано, то наверно лучше в параметр вынести?

**Comment by rgazo** [comment]
FwbModal is lacking the positioning ability that the original flowbite modal has.
